.github/workflows/automatic-test.yml

```diff
@@ -0,0 +1,40 @@
+# This workflow will create venv, install requirements & run nosetests.
+# The purpose is to run unit tests automatically when we push a remote branch to business-rule repo
+
+name: CI
+
+# Controls when the workflow will run
+#Triggers the workflow on push or pull request events but only for the master branch
+
+on: [push]
+
+
+  
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  test:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v3
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.8.10'
+          
+    # Installing all dependencies
+      - name: Install requirements
+        run: |
+          pip install -r dev-requirements.txt
+      
+    # running the tests
+      - name: Running Tests
+        run: |
+          cd tests
+          nosetests test_operators.py
```

Comment:
We need to run all tests here, not only test_operators

Reply:
Resolved.